lib/msf/core/payload/adapter/fetch.rb

```diff
@@ -0,0 +1,341 @@
+module Msf::Payload::Adapter::Fetch
+
+  def initialize(*args)
+    super
+    register_options(
+      [
+        Msf::OptEnum.new('FETCH_COMMAND', [true, 'Command to fetch payload', 'CURL', %w{ WGET CURL FTP TNFTP TFTP Certutil Psh_InvokeWebRequest }]),
```

Comment:
Seems like this should not be registered any more because the linux_options and windows_options mixin handles it with an appropriate value.

Reply:
fixed locally, will push again later.